package.json

```diff
@@ -43,9 +58,13 @@
     "js-yaml": "^3.14.0",
     "lint-staged": "^12.1.2",
     "prettier": "^2.2.1",
-    "ts-jest": "^26.4.4",
+    "ts-jest": "^26.4.2",
+    "ts-node": "^10.4.0",
     "typescript": "^4.1.3"
   },
+  "hooks": {
+    "pre-commit": "lint-staged && yarn build && git add dist/index.*"
+  },
```

Comment:
This is probably a merging error.

The pre-commit hook lives here now https://github.com/game-ci/unity-builder/blob/main/.husky/pre-commit

Reply:
Removed in latest